apis/openstack-heat/src/main/java/org/jclouds/openstack/heat/v1/domain/Attributes.java

```diff
@@ -0,0 +1,184 @@
+/**
+ * 
+ */
+package org.jclouds.openstack.heat.v1.domain;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.beans.ConstructorProperties;
+
+import javax.inject.Named;
+
+import org.jclouds.openstack.v2_0.domain.Link;
+import org.jclouds.openstack.v2_0.services.Image;
+
+import com.google.common.base.Objects;
+import com.google.common.base.Objects.ToStringHelper;
+
+/**
+ * @author deviyada
+ *
+ */
+public class Attributes {
+
+	@Named("id")
+	private String id;
+	@Named("progress")
+	private String progress;
+	@Named("status")
+	private String status;
+	@Named("config_drive")
+	private String config_drive;
+	@Named("tenant_id")
+	private String tenant_id;
+	@Named("hostId")
+	private String hostId;
+	@Named("created")
+	private String created;
+	@Named("name")
+	private String name;
+	@Named("user_id")
+	private String user_id;
+	@Named("links")
+	private Link[] links;
+
+	public String getProgress() {
+		return progress;
+	}
+
+	public void setProgress(String progress) {
+		this.progress = progress;
+	}
+
+	public String getStatus() {
+		return status;
+	}
+
+	public void setStatus(String status) {
+		this.status = status;
+	}
+
+	public String getConfig_drive() {
+		return config_drive;
+	}
+
+	public void setConfig_drive(String config_drive) {
+		this.config_drive = config_drive;
+	}
+
+	public String getTenant_id() {
+		return tenant_id;
+	}
+
+	public void setTenant_id(String tenant_id) {
+		this.tenant_id = tenant_id;
+	}
+
+	public String getHostId() {
+		return hostId;
+	}
+
+	public void setHostId(String hostId) {
+		this.hostId = hostId;
+	}
+
+	public Link[] getLinks() {
+		return links;
+	}
+
+	public void setLinks(Link[] links) {
+		this.links = links;
+	}
+
+	public String getId() {
+		return id;
+	}
+
+	public void setId(String id) {
+		this.id = id;
+	}
+
+	public String getCreated() {
+		return created;
+	}
+
+	public void setCreated(String created) {
+		this.created = created;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public String getUser_id() {
+		return user_id;
+	}
+
+	public void setUser_id(String user_id) {
+		this.user_id = user_id;
+	}
+
+	public Attributes build() {
+		return new Attributes(id, progress, status, config_drive, tenant_id, hostId, created, user_id, links);
+	}
+
+	@ConstructorProperties({ "id", "progress", "status", "config_drive", "tenant_id", "hostId", "created", "user_id",
```

Comment:
The order should be like in the constructor: for example user_id should be after created